interfaces/builtin/kubernetes_support.go

```diff
@@ -108,6 +108,8 @@ profile systemd_run (attach_disconnected,mediate_deleted) {
   @{INSTALL_DIR}/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/@{SNAP_REVISION}/{,usr/}bin/true ixr,
 ###KUBERNETES_SUPPORT_SYSTEMD_RUN###
 }
+
```

Comment:
could you add a comment about calico with at least a link to upstream or something?

Reply:
thanks